bluebottle/time_based/models.py

```diff
@@ -684,14 +684,14 @@ def event_data(self):
         }
 
 
-class DeadlineActivity(TimeBasedActivity):
+ONLINE_CHOICES = (
+    (None, 'Not set yet'),
+    (True, 'Yes, participants can join from anywhere or online'),
+    (False, 'No, enter a location')
+)
 
-    ONLINE_CHOICES = (
-        (None, 'Not set yet'),
-        (True, 'Yes, participants can join from anywhere or online'),
-        (False, 'No, enter a location')
-    )
 
+class BaseActivity(TimeBasedActivity):
```

Comment:
This seems odd... to have `BaseActivity` extending `TimeBasedActivity`... Couldn't these additions just be part of `TImeBasedActivity`? Or name this something that describes what you're adding?

Reply:
I do not really know what to do here. Maybe RegistrationActivity is the right word.